base/src/test/java/io/spine/logging/LoggingMethodTest.java

```diff
@@ -36,6 +36,7 @@
  * This test suite tests default methods of the {@link Logging} interface.
  */
 @DisplayName("Logging should provide a shortcut methods")
+@SuppressWarnings("deprecation") // ... and remove the class once deprecated API is removed.
```

Comment:
Do we have an issue for that? Maybe worth adding link to the issue?

Reply:
No need to have it since I'll be removing Slf4J dependency and this class won't pass compilation then.